glider_qc/glider_qc.py

```diff
@@ -500,7 +500,11 @@ def run_qc(config, ncfile, nc_path):
                              'flat_line_test',
                              'qc_rollup']:
 
-                qc_test = next(r for r in results if r.stream_id == var_name and r.test == testname)
+                try:
+                    qc_test = next(r for r in results if r.stream_id == var_name and r.test == testname)
```

Comment:
Always wrap functions which handle externally submitted data in an exception handler.  I always find new ways that user input fails so it's good to be defensive.